README.md

```diff
@@ -1,75 +1,83 @@
-|                 [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/)                 |            [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/honda0306)             |           [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/Mister-Corn)            |          [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/NandoTheessen)           |            [<img src="https://github.com/favicon.ico" width="15"> ](https://github.com/wvandolah)             |
-| [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/) | [ <img src="https://static.licdn.com/sc/h/al2o9zrvru7aqj8e1x2rzsrca" width="15"> ](https://www.linkedin.com/) |
 
-<br>
-<br>
+Tim Jeffries
+[<img src="https://avatars2.githubusercontent.com/u/48488539?s=400&u=af65e60066e644016e2d54d8e45cf86760e335df&v=4" width = "200" />](https://github.com/TimRexJeff)
 
-🚫 4️⃣ Optional examples of using images with links for your tech stack, make sure to change these to fit your project
 
-![MIT](https://img.shields.io/packagist/l/doctrine/orm.svg)
-![React](https://img.shields.io/badge/react-v16.7.0--alpha.2-blue.svg)
-![Typescript](https://img.shields.io/npm/types/typescript.svg?style=flat)
-[![Netlify Status](https://api.netlify.com/api/v1/badges/b5c4db1c-b10d-42c3-b157-3746edd9e81d/deploy-status)](netlify link goes in these parenthesis)
-[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
+Christopher Quinn
+[<img src="https://avatars1.githubusercontent.com/u/3174281?s=460&v=4" width = "200" />](https://github.com/xerillius)
 
-🚫 more info on using badges [here](https://github.com/badges/shields)
 
-## Project Overview
+Mary Clayton
+[<img src="https://avatars0.githubusercontent.com/u/52017102?s=460&u=3d6d943ad87af37a80ad0b80a67f2cfacee8c663&v=4" width = "200" />](https://github.com/mary-clayton)
+
 
-1️⃣ [Trello Board](🚫add link to trello board here)
 
-1️⃣ [Product Canvas](🚫add link to product canvas here)
 
-1️⃣ [UX Design files](🚫add link to design files here) - 🚫 delete if not applicable
+## Project Overview
 
-🚫 Replace lorem ipsum with a description of your project
+https://trello.com/b/bSVcWjZF/voicer
```

Comment:
I'd add some description to what each of these are and how we use them. "Trello is where we manage our development roadmap and track new features. It's a good place to start if you're looking to contribute." Or something along those lines. Same for the Notion doc link.

Reply:
absolutely, I'll see if I can get on that on the next revision